manifests/config.pp

```diff
@@ -52,12 +52,35 @@
     mode    => '0755',
     require => Class['jira::install'],
     notify  => Class['jira::service'],
-  } ->
+  }
+
+  if $jira::script_check_java_manage {
+    if $jira::script_check_java_template == undef {
+      $script_check_java_location = "puppet:///modules/${module_name}/check-java.sh"
+    } else {
+      $script_check_java_location = $jira::script_check_java_template
+    }
+    if (versioncmp($jira::version, '7.0.0') >= 0) {
+      file { "${jira::webappdir}/bin/check-java.sh":
+        source  => $script_check_java_location,
+        mode    => '0755',
+        require => [
+          Class['jira::install'],
+          File["${jira::webappdir}/bin/setenv.sh"],
+        ],
+        notify  => Class['jira::service'],
+      }
+    }
+  }
 
   file { "${jira::homedir}/dbconfig.xml":
     content => template("jira/dbconfig.${jira::db}.xml.erb"),
     mode    => '0600',
-    require => [ Class['jira::install'],File[$jira::homedir] ],
+    require => [
```

Comment:
This was needed because of my removal of `->` after the `setenv.sh` script management. I don't recall why I decided to put the code block here, so I'll revert this back.